oap-server/server-bootstrap/src/main/resources/application.yml

```diff
@@ -66,7 +66,7 @@ core:
     enableDataKeeperExecutor: ${SW_CORE_ENABLE_DATA_KEEPER_EXECUTOR:true} # Turn it off then automatically metrics data delete will be close.
     dataKeeperExecutePeriod: ${SW_CORE_DATA_KEEPER_EXECUTE_PERIOD:5} # How often the data keeper executor runs periodically, unit is minute
     recordDataTTL: ${SW_CORE_RECORD_DATA_TTL:3} # Unit is day
-    metricsDataTTL: ${SW_CORE_RECORD_DATA_TTL:7} # Unit is day
+    metricsDataTTL: ${SW_CORE_METRICS_DATA_TTL:7} # Unit is day
```

Comment:
FYI @innerpeacez Notice, this change should be included in the helm next release.

Reply:
This env variable is not used in the chart, set according to the user's own needs, chart does not need to be added.